docs/app_metrics_guide.adoc

```diff
@@ -0,0 +1,81 @@
+
+[discrete]
+== Prerequisites
+
+* An App that is using the AeroGear SDK (link TODO)
+* A running Metrics Service (link TODO)
+* An OpenShift account that will provide access to Grafana in the Metrics Service (link TODO)
+
+[discrete]
+== Accessing visualisations
+
+There are many visualisations available in the default dashboards in Grafana.
+There are 2 dashboards where App Metrics data is shown:
+
+* Mobile Service Dashboard (Summary data in the 'Mobile App Metrics' section)
+* Mobile App Metrics Dashboard
+
+In general, the summary data that is visible on the 'Mobile Services' Dashboard is also available on the 'Mobile App Metrics' dashboard, but in more detail.
+
+[discrete]
+== Analyzing App launches
+
+An App launch is a metric event that occurs when an App is launched on a device.
+The total number of App launches for a given time shows how many times an App was launched in that time, regardless of if it was the same device or different devices.
+It can be useful as a gauge on how much activity the App has.
+However, as it doesn't take into account multiple lauches from the same device, it isn't useful as a gauge for 'stickiness' of the App.
+
+[discrete]
+== Analyzing unique clients
+
+Every metric event has a unique client identifier associated with it.
+This unique client identifier is generated the first time an App is started.
+It is worth noting this identifier will be regenerated if a user uninstalls and re-installs an App.
+The total number of unique clients for a given time shows how many users launched an App at least once.
+This can be useful as a gauge on how many active App users there currently are.
+The time range is an important factor when viewing the number of unique clients.
+Viewing unique clients for the last year gives a very different insight than for the last week.
+In Grafana, the time range can be changed in the top right of the Dashboard view.
+
+[discrete]
+== Analyzing unique clients per App
+
+Every metric event has an App identifier associated with it.
+An App identifier uniquely identifies an App.
+It is bundled with the App binary.
+There can be more than 1 App (each with a different App identifier) sending metrics events to the same Metrics Service.
+For example, there might be an Admin Portal App and a Field Worker App both using the same backend services and same Metrics Service.
+The number of unique clients per App gives a breakdown of all users by the App they are using.
+However, if you only have 1 App there is no difference between the number of unique clients and the number of unique clients per App.
+
+[discrete]
+== Analyzing unique clients per platform
+
+Every metric event has the platform associated with it.
+An example of platform is Android or iOS.
+This can be useful for making decisions about which platform to put effort into.
+However, if you only have 1 App on 1 platform (e.g a single Android App), or you have multiple Apps all on the same platform (e.g. 2 Android Apps)
```

Comment:
I think we can just say "If you have all your apps on a single platform, there is no diff..."